parsing_scripts/transactions.rb

```diff
@@ -0,0 +1,58 @@
+require 'pry'
+require 'nokogiri'
+require 'open-uri'
+
+
+class Transactions
+
+  def initialize (browser)
+   @browser = browser
+  end
+
+  def get_transactions_info
+    go_to_transactions
+    transactions = [] 
+    extract_data(transactions)
+  end
+
+  def go_to_transactions
+    @browser.li(class:"tr_history-menu-item").click
+  end
+  
+  def get_html
+    @browser.div(class: "day-header").wait_until(&:present?)
+    Nokogiri::HTML.fragment(@browser.div(class:"operations").html)
+  end
+
+  def extract_data (transactions)
+    @html = get_html
+    divs_count = @html.search("div").size
+    
+    div_index = 1 
```

Comment:
if you will use .each you will not need this variable any more

Reply:
Fixed